communities/massmesh/meshnode/files/etc/config/system

```diff
@@ -0,0 +1,19 @@
+config system
+    option hostname 'OpenWrt'
+    option ttylogin '1'
```

Comment:
unknown what this does, but added it.. no serial on boot could be re: askfirst entry